tests/std/tests/GH_000690_overaligned_function/test.cpp

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <cassert>
+#include <cstddef>
+#include <cstdint>
+#include <functional>
+
+#ifndef __cpp_aligned_new
+#error This test is only valid when C++17 over-aligned allocation is supported
+#endif
+
+#pragma warning(disable : 4324) // structure was padded due to alignment specifier
+
+// SFO should not happen

```

Comment:
This test repeatedly refers to "SFO" which we don't usually abbreviate; expanding the acronym on first use is sufficient.
```suggestion
// SFO (Small Functor Optimization) should not happen
```